assets/src/edit-story/app/media/media3p/attribution.js

```diff
@@ -77,7 +81,7 @@ export function CoverrAttribution() {
     <a href={coverrUrl} target={'_blank'} rel={'noreferrer'}>
       <AttributionPill>
         {__('Powered by', 'web-stories')}
-        <CoverrLogoSpan>{'COVERR'}</CoverrLogoSpan>
+        <CoverrLogo>{'COVERR'}</CoverrLogo>
```

Comment:
This should just be `<CoverrLogo />`. The 'COVERR' text is not used.

Also, why is the Coverr logo way bigger than the Unsplash logo?

Reply:
Addressed in https://github.com/google/web-stories-wp/pull/4011

The Coverr logo looks much bigger due to their font size.